README-agents.md

```diff
@@ -73,12 +73,14 @@ make trigger-pipeline JIRA_ISSUE=RHEL-12345
 # Test specific agents standalone
 make JIRA_ISSUE=RHEL-12345 run-triage-agent-standalone
 make PACKAGE=httpd VERSION=2.4.62 JIRA_ISSUE=RHEL-12345 BRANCH=c10s run-rebase-agent-standalone
-make PACKAGE=httpd UPSTREAM_FIX=https://github.com/... JIRA_ISSUE=RHEL-12345 BRANCH=c10s run-backport-agent-standalone
+make PACKAGE=httpd UPSTREAM_PATCHES=https://github.com/... JIRA_ISSUE=RHEL-12345 BRANCH=c10s run-backport-agent-standalone
 
 # Or with dry-run
 DRY_RUN=true make JIRA_ISSUE=RHEL-12345 run-triage-agent-standalone
 ```
 
+Variable `UPSTREAM_PATCHES` is comma separated in case you have multiple upstream commits as an input.
```

Comment:
```suggestion
Use commas to delimit multiple patch/commit URLs in `UPSTREAM_PATCHES`.
```

It could also be enough to do e.g. `make PACKAGE=httpd UPSTREAM_PATCHES=https://example.com/patch1,https://example.com/patch2` in the example.